src/endpoints/http/collections/create_test.js

```diff
@@ -2,6 +2,13 @@ import supertest from 'supertest';
 import express from 'express';
 import bodyParser from 'body-parser';
 import {createCollection} from './create.js';
+import authorize from '../../../middleware/route-authorize';
+
+authorize.config = {
+  globalPermissionPath: 'user.permission',
+  globalAccessPrefix: 'user.access.',
```

Comment:
maybe it would be better to not require the `.` at the end here. Its easy to forget. We can add the dot when we concatenate the prefix with the rest.